libvlc-sys/build.rs

```diff
@@ -18,9 +17,15 @@ fn main() {
         .whitelist_function(".*vlc.*")
         .whitelist_var(".*vlc.*")
         .whitelist_function("vsnprintf")
-        .parse_callbacks(Box::new(bindgen::CargoCallbacks))
-        .generate()
-        .expect("Unable to generate bindings");
+        .parse_callbacks(Box::new(bindgen::CargoCallbacks));
+
+    // Set header include paths
+    let pkg_config_library = pkg_config::Config::new().probe("libvlc").unwrap();
+    for include_path in &pkg_config_library.include_paths {
+        bindings = bindings.clang_arg(format!("-I{}", include_path.display()));
```

Comment:
Hi, I'm not sure `include_path.display()` is correct here, it seems that it won't work with spaces and non-unicode characters: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=5f4f47e8706b68ed348db726f292ba1e

I'm not sure of what should be used though.

The available variables with pkg-config are:

```
libs: Vec<String>
link_paths: Vec<PathBuf>
frameworks: Vec<String>
framework_paths: Vec<PathBuf>
include_paths: Vec<PathBuf>
defines: HashMap<String, Option<String>>
version: String
```

On Linux, only `Libs` and `includedir` are filled in the pc file. Here include_path is correctly set but libs are not. It should probably used the additional keys from [1]. In addition, it's probably missing the MacOSX part which has framework containing both the includes and the libs. Do you have a MacOSX to do this or would you like me to experiment and determine what's needed ?

[1]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

Reply:
This include path usage is based on sdl2-sys.

https://github.com/Rust-SDL2/rust-sdl2/blob/master/sdl2-sys/build.rs

And `.probe("libvlc")` will print 

```
cargo:rustc-link-search=native=/usr/lib/x86_64-linux-gnu
cargo:rustc-link-lib=vlc
```

I don't know about MacOSX framework, so PR for it will be great help.